### ISSUE TITLE
StockChart containing Column series ignoring series tooltip template.

### ISSUE THREAD

**Issue by ncrawlins** [telerik/kendo-angular#791]
## I'm submitting a...

* Bug report 
-- Not sure if this is a bug or I'm doing it wrong

## Current behavior
Chart tooltips based on a template in a stockChart aren't showing properly

## Expected behavior
proper tooltip based on template

## Minimal reproduction of the problem with instructions
the following tooltip config inside a column series works fine in a standard chart...

```
<kendo-chart-series-item-tooltip>
  <ng-template let-category="category" let-series="series" let-value="value">
    <div>{{category | date:'shortDate'}}</div>
    <div>{{series.name}}</div>
    <div>{{value}}</div>
  </ng-template>
</kendo-chart-series-item-tooltip>
```
But I changed my chart to a stockchart so that I could have a navigator at the bottom. In a stockchart the column series and navigator, seem to work except for the tooltip template no longer has any impact.  it just shows a value.

## What is the motivation / use case for changing the behavior?
column chart with navigator and custom tooltips.

## Environment

Package versions:

```
+-- @angular/animations@4.3.2
+-- @angular/cdk@2.0.0-beta.8
+-- @angular/cli@1.3.0-rc.3
+-- @angular/common@4.3.2
+-- @angular/compiler@4.3.2
+-- @angular/compiler-cli@4.3.2
+-- @angular/core@4.3.2
+-- @angular/forms@4.3.2
+-- @angular/http@4.3.2
+-- @angular/material@2.0.0-beta.8
+-- @angular/platform-browser@4.3.2
+-- @angular/platform-browser-dynamic@4.3.2
+-- @angular/router@4.3.2
+-- @progress/kendo-angular-charts@1.2.2
+-- @progress/kendo-angular-dateinputs@1.0.5
+-- @progress/kendo-angular-dropdowns@1.1.2
+-- @progress/kendo-angular-excel-export@1.0.3
+-- @progress/kendo-angular-grid@1.2.3
+-- @progress/kendo-angular-inputs@1.0.7
+-- @progress/kendo-angular-intl@1.2.2
+-- @progress/kendo-angular-l10n@1.0.2
+-- @progress/kendo-data-query@1.0.6
+-- @progress/kendo-drawing@1.1.2
+-- @progress/kendo-theme-default@2.38.2
+-- @types/node@8.0.17
+-- core-js@2.4.1
+-- hammerjs@2.0.8
+-- rxjs@5.4.2
+-- typescript@2.4.2
`-- zone.js@0.8.16
```

Browser:

- Chrome (desktop) version 60

System:
- TypeScript version: 2.4.2
- Node version: 8.2.1 Npm: 5.3
- Platform:  Windows 10

**Comment by ncrawlins** [comment]
[Plunkr demonstration](http://plnkr.co/edit/EYhfb0HaW0knrJoiEPXl?p=preview)
This plunker shows 2 stock charts, the only difference is one has a navigator and one doesn't, but they display the tooltips on the chart differently.

**Comment by tsvetomir** [comment]
Thanks for the bug report. I re-formatted [the plunkr](http://plnkr.co/edit/s3asrfqpH2rMwZqm0vlB?p=preview) for clarity. Indeed, adding the navigator disables the tooltip template.

**Comment by siyka-marinova** [comment]
Fix is available in `@progress/kendo-angular-charts@v1.2.3`
